Adding new documentation format and openning the user-guide for more contributions that will come soon (#368)
openshift/assisted-service: README.md

    * kubeadmin-password

   Files are uploaded to deployed S3 under the name template "<cluster-id>/<filename>".

## Documentation

> To add modifications and sections on the documentation, we are using [MkDocs](https://www.mkdocs.org/), to install the dependencies, just execute this:

Reviewer: ```suggestion
We are using [MkDocs](https://www.mkdocs.org/) to build the documentation.  It can be installed by executing:
```

```shell
pip3 install --user mkdocs
```

> - If you add a new section, modify the `mkdocs.yaml` 

Reviewer: ```suggestion
- When adding a new section, modify the `mkdocs.yaml` file accordingly.
```

> - If you modify an existant one, test it with 

Reviewer: ```suggestion
- When you modifying an existing one, test it by executing:
```
